serverless.yml

```diff
@@ -61,6 +61,27 @@ functions:
           path: test-results/delete
           method: put
 
+  passCertificateDuplicates:
```

Comment:
Can we update to reflect name of lambda i.e. `putPassCertificateDuplicates` and update the handler path

Same for all 3 new lambdas

Reply:
This https://github.com/dvsa/mes-result-service/pull/297#issuecomment-1635493926 should probably be addressed first, and then update the `serverless.yml`